webapp/app.py

```diff
@@ -362,6 +366,63 @@ def search_docs():
     )
 
 
+CACHE_TTL = 60 * 60  # 1 hour cache
+
+
+@app.route("/juju/latest.json")
+@cross_origin()
+@cached(cache=TTLCache(maxsize=128, ttl=CACHE_TTL))
+def get_latest_versions():
+    try:
+        result = {}
+
+        # get dashboard version
+        dashboard_response = requests.get(
+            "/".join(
+                [
+                    "https://api.github.com",
+                    "repos",
+                    "canonical",
+                    "jaas-dashboard",
```

Comment:
```suggestion
                    "juju-dashboard",
```

I'm assuming the repo was renamed at some point. Github does automatically handle the redirect for us from https://github.com/canonical/jaas-dashboard to https://github.com/canonical/juju-dashboard, but it could be changed here if you're feeling like it.

Reply:
Thanks Tom! I've updated the repo name